src/permissions/PermissionController.test.ts

```diff
@@ -4042,10 +4030,9 @@ describe('PermissionController', () => {
 
       const controller = getDefaultPermissionController(options);
 
-      await expect(
-        async () => await controller.rejectPermissionsRequest(id),
-      ).rejects.toThrow(new errors.PermissionsRequestNotFoundError(id));
+      await controller.rejectPermissionsRequest(id);
 
+      // Method is called only once
```

Comment:
Same thing here as above.

Reply:
Edit: In case it wasn't clear I just mean since the test says "it does not call ApprovalController:rejectRequest" then that should be tested here as well.